spec/schema.json

```diff
@@ -15,15 +15,97 @@
     }
   ],
   "paths": {
-    "/transactions/{transactionId}/limited-partnership/partnership": {
+    "/transactions/{transaction_id}/incorporation/limited-partnership": {
+      "post": {
+        "tags": [
+          "Limited Partnerships"
+        ],
+        "summary": "Create a Limited Partnership incorporation",
+        "parameters": [
+          {
+            "name": "transaction_id",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Limited Partnership Incorporation has been successfully created.",
+            "schema": {
+              "$ref": "#/definitions/CreatedResponse"
+            }
+          },
+          "400": {
```

Comment:
Mentioned on the spec but not sure how a 400 could be returned. There's no payload, just ids in the request.

Reply:
👍 happy to leave this in.